doc/source/user_guide/custom_operators.rst

```diff
@@ -78,32 +78,32 @@ This example script shows how you create a basic operator plugin:
             record_operator(CustomOperator, *args)
 
 
-In the various properties for the class, you specify the following:
+In the various properties for the class, specify the following:
 
 - Name for the custom operator
 - Description of what the operator does
-- Dictionary for each input and output pin, which includes the name, a list of supported types, a description,
+- Dictionary for each input and output pin. This dictionary includes the name, a list of supported types, a description,
   and whether it is optional and/or ellipsis (meaning that the specification is valid for pins going from pin
   number *x* to infinity)
 - List for operator properties, including name to use in the documentation and code generation and the
-  operator category. The optional ``license`` property allows to define a required license to check out
+  operator category. The optional ``license`` property allows you to define a required license to check out
```

Comment:
```suggestion
  operator category. The optional ``license`` property lets you define a required license to check out
```
Using "lets you define...." is easier to process than "allows you to define...."